cpp/oneapi/dal/algo/decision_forest/backend/gpu/infer_train_reg_kernel_dpc_test.cpp

```diff
@@ -17,8 +17,8 @@
 #include <CL/sycl.hpp>
```

Comment:
It's not about PCA

Reply:
I've added support for DPC++ tests in Bazel to be able to run DPC++ PCA tests and had to fix existing rules/tests.